.github/workflows/changelog.yaml

```diff
@@ -0,0 +1,34 @@
+name: Generate Changelog for Porter
+on:
+  release:
+    types: [published]
+
+jobs:
+  changelog:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3.3.0
+        with:
+          ref: main
+          fetch-depth: 0
+          token: ${{ secrets.GH_PAT }}
```

Comment:
Should we be using [this instead](https://github.com/getporter/porter/blob/main/.github/workflows/prow-merge-pr.yaml#L17) - otherwise looks great!